doc/man1/openssl-verify.pod.in

```diff
@@ -336,45 +337,45 @@ determined.
 
 The verify operation consists of a number of separate steps.
 
-Firstly a certificate chain is built up starting from the supplied certificate
-and ending in the root CA.
-It is an error if the whole chain cannot be built up.
-The chain is built up by looking up the issuers certificate of the current
-certificate.
-If a certificate is found which is its own issuer it is assumed to be the root
-CA.
-
-The process of 'looking up the issuers certificate' itself involves a number of
-steps.
-After all certificates whose subject name matches the issuer name of the current
-certificate are subject to further tests.
+Firstly a certificate chain is built up starting from the target certificate
+and typically ending in a self-signed "root" CA certificate. It is an error if
+the whole chain cannot be built up unless the B<-partial_chain> option is given.
+The chain is built up step by step, looking up the issuer's certificate of the
+current certificate. If a certificate is found that appears to be its own issuer
+it is assumed to be the self-signed root, which must be trusted.
+
+The process of 'looking up the issuer's certificate' itself involves a number of
+steps. All available certificates with a subject name that matches the issuer
+name of the current certificate are subject to further tests.
 The relevant authority key identifier components of the current certificate (if
 present) must match the subject key identifier (if present) and issuer and
-serial number of the candidate issuer, in addition the keyUsage extension of
-the candidate issuer (if present) must permit certificate signing.
+serial number of the candidate issuer certificate.
 
-The lookup first looks in the list of untrusted certificates and if no match
-is found the remaining lookups are from the trusted certificates. The root CA
-is always looked up in the trusted certificate list: if the certificate to
-verify is a root certificate then an exact match must be found in the trusted
-list.
+The lookup first uses the list of non-trusted "intermediate" CA certificates and
+if no match is found the remaining lookups are from the trusted certificates.
+The root CA is always looked up in the trusted certificate list:
+if the certificate to verify is a self-signed certificate
+then an exact match must be found in the trust store.
```

Comment:
Good that we've come across those outdated passages in `openssl-verify.pod.in`. 
I've updated them accordingly and also tweaked the above quoted section of `X509_VERIFY_PARAM_set_flags.pod` to make immediately clear what the current situation is (since v1.1.0).

Reply:
Aargh.  Please see #10132   I *hope* that PR of mine gets done before this one.

Reply:
A pity that that PR was idle for quite some time - if it had been merged already I would have built on it. I just learned about it through your comment.
It looks like there are overlaps only in the paragraphs of the section starting
```
The verify operation consists of a number of separate steps.
```
So merging (either way) luckily should not be much effort.